ocs_ci/ocs/ocp.py

```diff
@@ -1470,6 +1471,89 @@ def wait_for_cluster_connectivity(tries=200, delay=3):
     return retry(CommandFailed, tries=tries, delay=delay, backoff=1)(service.get)()
 
 
+def wait_for_job_completion(job_name, namespace, timeout=600, sleep_time=30):
```

Comment:
Please move job specific functions to [ocs_ci/ocs/resources/job.py](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/job.py)

Reply:
Ook.